src/containers/editor/graph/Toolbar.tsx

```diff
@@ -24,6 +24,8 @@ const Toolbar = memo(({ id }: ToolbarProps) => {
   const { getNodes, getEdges, setNodes, setEdges } = useReactFlow<NodeWithData, EdgeWithData>();
   const nodes = getNodes();
   const edges = getEdges();
+  const hasSiblings = edges.some((edge) => edge.source === id);
```

Comment:
`edges` are only part of the full set of edges when there are too many nodes (see `computeVisible` in `src/lib/graph/computeVisible.ts`), I think we need to [add `graph` back into `treeStore`](https://github.com/loggerhead/json4u/commit/fe43579e46dcec84d42c26d897bcd102bb587c77#diff-8488a892db44fd87d5587a0be4ad12934e34569cf99450d994883a9ebff04fc5L58), and then use it here.

Reply:
let me take a look at this.

Reply:
Is your suggestion to replace `useReactFlow` by `useNodesAndEdges` in the component?

Reply:
I meant to say that you can use `useTreeStore((state) => state.graph)`, which was deleted by me. You need to add it back. But it's okay, you can leave it to me to fix it.

Reply:
This is reverted. Thank you.